linode/manager
# test: [M3-8542] - Add unit tests for `CopyableTextField` component

## Description 📝
Unit test cases for `CopyableTextField` components.

## Changes  🔄

- Add unit test cases for `CopyableTextField` components.

## Target release date 🗓️
N/A

## How to test 🧪

### Verification steps
(How to verify changes)
- Run `yarn test src/components/CopyableTextField --reporter verbose`
- Verify that all the test cases pass.

## As an Author I have considered 🤔

*Check all that apply*
